tool/test/org/antlr/v4/test/rt/gen/Generator.java

```diff
@@ -597,6 +601,10 @@ private JUnitTestFile buildParserErrors() throws Exception {
 				"ad",
 				"",
 				"line 1:1 missing {'b', 'c'} at 'd'\n");
+		file.addParserTest(input, "SingleSetInsertionConsumption", "T", "a",
```

Comment:
What was the output of this test using the old code generation? It seems similar to the `SingleTokenInsertion` test, so I wanted to make sure it's actually capable of detecting the error.

My instinct was the best way to test this is looking at both the error message output and the resulting parse tree, but if this can detect the bug in the old code then I'm ok with it.